designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,302 @@
+# `dotnet nuget verify --verbosity`
+
+* Status: **In Review**
+* Author: [Kartheek Penagamuri](https://github.com/kartheekp-ms)
+* Issue: [#10316](https://github.com/NuGet/Home/issues/10316) - dotnet nuget verify is too quiet
+
+## Problem background
+
+`dotnet nuget verify` command doesn't provide any output by default due to logical error in the implementation. If the user didn't pass value for `--verbosity` option, then the log level is set to [`LogLevel.Minimal`](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L58). Customers need information about packages’ signatures to [manage package trust boundaries](https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages).
```

Comment:
Regarding the last sentence, customers using `dotnet nuget verify` to configure client trust policies, is this the primary or secondary/edge use case?

If it's the primary use case, it seems to me this is hard to discover and improving the command name to something like `get-signature` would help with discoverability, and expectations for what default output should be.

Otherwise, `dotnet nuget verify` sounds like the primary scenario is check if the package signature is valid or not, hence a single line of output per package would be the expected output at default verbosity.

I have no idea how customer are using `nuget.exe verify` or `dotnet nuget verify` "in the real world".

Reply:
As you have mentioned above, the primary use case of this command is signature verification. Configuring client trust policies is the `secondary/edge use case`. I like your idea for a new command just to obtain signatures for configuring the client trust policies. copying @JonDouglas just in case if there has been customer ask around this request.

May not be relevant here but @dtivel recently added a tool to extract certificates from signed NuGet package https://github.com/NuGet/Entropy/pull/45